examples/server_hardware.py

```diff
@@ -47,7 +47,7 @@
 
 # Set the server_hardware_id to run this example.
 # server_hardware_id example: 37333036-3831-4753-4831-30315838524E
-server_hardware_id = "34323937-3431-4732-3230-313130364747"
+server_hardware_id = "30303437-3034-4D32-3230-313133334752"
```

Comment:
I see there was already some info filled here, but if we're being thorough, I'd say the best is for this to be left empty and be filled by the user if the user wishes to use it.
```python
server_hardware_id =''
```
Then on the actions that will make use of the server hardware id, put in a `if server_hardware_id:` so that the actions are only executed if this has been filled out.

Reply:
Sure will do. But most of the examples have some of the variables assigned with values this way. If changed here, we would have to change in all the other examples too, right?